nature/styleguide/typography.md

```diff
@@ -129,3 +155,10 @@ Use our font styles consistently by always making sure they reflect the informat
         </tr>	
 </table>	
 
+### Changing default heading sizes
+
+Our typography styles determine the size of headings based on the heading level you apply. For example, an `<h1>` is the largest at 30px, followed by `<h2>` at `24px`, and so on.
+
+If you need to use a different font size for a heading, you can use a utility class to override the default sizing.
```

Comment:
Does this work?

```suggestion
If you need to use a different font size for a heading, you can use a utility class to override the default sizing. For example:

`<h2 class="u-h3">Smaller h2 rendering</h2>`
```

Reply:
Will need to be a code block, but yes! Like this:

```
```
<h2 class="u-h3">Smaller h2 rendering</h2>
```
```

Reply:
(excuse the backslashes, I'm trying to escape it and failing)

Reply:
OK cool - I'll update. Thanks!